packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -1640,30 +1644,34 @@ class _TableRowState<T> extends State<TableRow<T>>
       }
     }
 
-    final rowContent = Padding(
-      padding: const EdgeInsets.symmetric(horizontal: defaultSpacing),
-      child: ListView.builder(
-        scrollDirection: Axis.horizontal,
-        controller: scrollController,
-        itemCount: widget.columns.length + widget.columns.numSpacers,
-        itemBuilder: (context, int i) {
-          final displayTypeForIndex = rowDisplayParts[i];
-          switch (displayTypeForIndex) {
-            case _TableRowPartDisplayType.column:
-              final index = columnIndexMap[i]!;
-              return columnFor(
-                widget.columns[index],
-                widget.columnWidths[index],
-              );
-            case _TableRowPartDisplayType.columnSpacer:
-              return const SizedBox(
-                width: columnSpacing,
-                child: VerticalDivider(width: columnSpacing),
-              );
-            case _TableRowPartDisplayType.columnGroupSpacer:
-              return const _ColumnGroupSpacer();
-          }
-        },
+    final rowContent = MouseRegion(
+      onEnter: (_) => setState(() => isHovering = true),
+      onExit: (_) => setState(() => isHovering = false),
```

Comment:
adding a MouseRegion for every row may be expensive. We have hit performance issues in the past by doing stuff like this (anything that adds a bunch of listeners). Instead of adding a MouseRegion for each row, add a single MouseRegion for the _Table widget and then calculate the hovered rowed based on the event's dy value.

Example: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/shared/charts/flame_chart.dart/#L283

Reply:
Or another option is to make hover handling opt-in for the table widget, so that this `MouseRegion` is only added for the deep links table. At least then we can limit the performance impact and address this if it becomes a performance concern for the deep links table.

Reply:
changed it to opt-in